src/engine/video/video.cpp

```diff
@@ -389,8 +405,7 @@ const std::string VideoEngine::CreateGLErrorString()
 
     if(error_string == nullptr)
         return ("Unknown GL error code: " + NumberToString(_gl_error_code));
-    else
-        return (char *)error_string;
+    return static_cast<char*>(error_string);
```

Comment:
Ah indeed. It is probably because of the const and the unsigned char* combined.
`return reinterpret_cast<const char*>(error_string);` will work :)